src/conn/conn_api.c

```diff
@@ -2626,14 +2626,35 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler,
 
 	WT_ERR(__wt_config_gets(session, cfg, "file_extend", &cval));
 	for (ft = file_types; ft->name != NULL; ft++) {
+		/*
+		 * An unset log extend length is an indicator to set it
+		 * with configured maximum log file size in case of the default
+		 * configuration. Update the log extend length with actual
+		 * desired size when maximum log file size gets initialized as
+		 * part log server initialization.
+		 */
+		conn->log_extend_len = WT_CONFIG_UNSET;
 		ret = __wt_config_subgets(session, &cval, ft->name, &sval);
 		if (ret == 0) {
 			switch (ft->flag) {
 			case WT_DIRECT_IO_DATA:
 				conn->data_extend_len = sval.val;
 				break;
 			case WT_DIRECT_IO_LOG:
-				conn->log_extend_len = sval.val;
+				/*
+				 * In the case of the "file_extend=(log=)",
+				 * unset the log extend length.
```

Comment:
I suggest a minor change to this comment:
* When using "file_extend=(log=)", the val returned is 1.
* Unset the log extend length in that case to use the default.

Reply:
Done.